web/src/middleware/api-version.ts

```diff
@@ -0,0 +1,28 @@
+/*
+* CORTX-CSM: CORTX Management web and CLI interface.
+* Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published
+* by the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com or cortx-questions@seagate.com.
+*/
+import { Request, Response, NextFunction } from "express";
+import { HTTP400Error } from "../utils/http-errors";
+
+const apiVersions = ["v1"]
+
+export const checkApiVersion = (req: Request, res: Response, next: NextFunction): void => {
+  if (apiVersions.includes(req.params.version)) {

```

Comment:
For now use simple switch statement to check for v1 and v2 instead of array search. Add a comment on this function mentioning that as there is very less chance that there will be more versions, hence list is not required. In future if there are more version then we will use list.